<launch>

  <rosparam file="$(find robot_arm_control)/config/robot_arm_control.yaml" command="load"/>

  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
    output="screen" ns="/robot_arm" args="joint1_position_controller joint2_position_controller joint3_position_controller joint4_position_controller joint5_position_controller joint6left_position_controller joint7left_position_controller joint8left_position_controller joint9right_position_controller joint10right_position_controller joint11right_position_controller joint_state_controller"/>

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
    respawn="false" output="screen">
    <remap from="/joint_states" to="/robot_arm/joint_states" />
  </node>

</launch>